entrypoint

```diff
@@ -160,6 +160,16 @@ else
     # make the dump
     mysqldump -h $DBSERVER -u$DBUSER -p$DBPASS $DB_LIST | gzip > ${TMPDIR}/${TARGET}
 
+    # Execute additional scripts for post porcessing. For example, create a new
+    # backup file containing this db backup and a second tar file with the
+    # contents of a wordpress install.
+    if [ "$(ls -A /scripts.d/post-backup/)" ];
+    then
+      for i in $(ls /scripts.d/post-backup/*.sh);do
+          source $i
```

Comment:
Well, there are many variables that could be useful on post processing, I would not know which ones should be exposed and which ones not. 

The idea of doing that is to execute them as an extension of the main script instead of a separate process which requires more code to pass variables, do input validations, handle exit codes, etc. 

What's your issue of sourcing them ? I do not see the potential issues with doing it.